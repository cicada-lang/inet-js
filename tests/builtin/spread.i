type Nat -- @Type end
node zero -- Nat :value! end
node add1 Nat :prev -- Nat :value! end

node add
  Nat :target!
  Nat :addend
  --------
  Nat :result
end

(add) @spread $target $addend $result
target @inspect
addend @inspect
result @inspect
