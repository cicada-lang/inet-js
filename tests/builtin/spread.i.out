netFromPort (add₀)-target! end
netFromPort (add₀)-addend end
netFromPort (add₀)-result end
